scripts/install-git-hooks.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Here is my sample:

```
#!/usr/bin/env bash

add_hook () {
  WORKING_DIR=$PWD
  GIT_DIR=".git/hooks"
  SCRIPT_DIR="../../scripts/git-hooks"

  # Change our working directory.
  cd ${GIT_DIR}

  # Remove old hooks (the file and the link).
  [ -e ${1} ] && rm ${1}
  [ -L ${1} ] && rm ${1}

  # Ensure we have set all scripts as executable.
  cp "${SCRIPT_DIR}/${1}.sh" "./${1}"
  chmod 777 "./${1}"
  chmod +x "./${1}"

  # Change the working directory back.
  cd ${WORKING_DIR}
}

add_hook commit-msg
add_hook pre-commit
add_hook pre-push
```

Reply:
I've applied some changes based on this snippet, thanks!